common:        
   pcd_save_en: false  
   Save_path: true
   ROOT_DIR: "Onion-LO/src/onion_lo/results/"
   save_frame_num_beg: 1
   save_frame_num_end: 1600
   lidar_topic: "/points_raw"
   child_frame: "base_link"
   odom_frame: "odom"

kiss_l:
   voxel_size: 2
   lidar_type: "KITTI"
   Deskew: true
   exp_key_num: 1000
   Resolution_h: 0.2
   Resolution_v: 0.42
